setup.py

```diff
@@ -33,7 +33,8 @@
     include_package_data=True,
     setup_requires=[
         "setuptools_scm<6.0.0; python_version <= '3.6'",
-        "setuptools_scm; python_version > '3.6'",
+        "setuptools_scm<8.0.0; python_version < '3.8'",
+        "setuptools_scm; python_version >= '3.8'",
```

Comment:
This should not be necessary has setuptools_scm 8 declares it needs python >= 3.8, so pip will not pick it and use a lower version with python 3.7. Unless you have a very old pip ?

Reply:
i am using docker to migrate a database from 13 to 14, i dont know if that is a workflow supported, but the pip i am using is the pip is installed with docker:14.0